guides/deploying/fly.md

```diff
@@ -32,19 +31,30 @@ ip_filter = Application.get_env(:your_app, :ice_ip_filter)
 In `runtime.exs`:
 
 ```elixir
-if System.get_env("FLY_IO") do
+if System.get_env("FLY_APP_NAME") do
   config :your_app, ice_ip_filter: &ExWebRTC.ICE.FlyIpFilter.ip_filter/1
 end
 ```
 
-In fly.toml:
+Now:
+1. Run `fly launch`. It will generate a Dockerfile that will fail to build.
+2. Introduce the following changes 
 
-```toml
-[env]
-  # add one additional env
-  FLY_IO = 'true'
-```
+    ```diff
+    - ARG ELIXIR_VERSION=1.16.0
+    - ARG OTP_VERSION=26.2.1
+    - ARG DEBIAN_VERSION=bullseye-20231009-slim
+    + ARG ELIXIR_VERSION=1.17.2
+    + ARG OTP_VERSION=27.0.1
+    + ARG DEBIAN_VERSION=bookworm-20240701-slim
+
+    - RUN apt-get update -y && apt-get install -y build-essential git \
+    -     && apt-get clean && rm -f /var/lib/apt/lists/*_*
+    + RUN apt-get update -y && apt-get install -y build-essential git pkg-config libssl-dev \
+    +     && apt-get clean && rm -f /var/lib/apt/lists/*_*
```

Comment:
We could mention that it may be necessary to add `libsrtp2-dev` as well -- notably when building on an arm64 machine (or targeting arm64 Linux in general, seeing as `ex_libsrtp` doesn't provide a prebuilt package in that case)